test.py

```diff
@@ -46,13 +45,50 @@ def get_passwords_basic():
 
 
 if __name__ == '__main__':
-    folder_id = "1d932dc0-d0a3-4a44-80c7-4701f84dc307"
-    with passboltapi.PassboltAPI(config_path="config.ini") as passbolt:
-        # required: passbolt.import_public_keys() when the folder has more users.
-        print(passbolt.create_resource(
-            name='Sample Name',
+    
+    now = datetime.now() # current date and time
+
+    parent_folder_id = "5b024c98-211c-487e-86f1-0bf75de3f685"
+
+    new_user_name = f"passbolt-py-user-%s@atexo.com"%(now.strftime("%Y-%m-%d-T%H%M%S"))
```

Comment:
Please avoid adding any personal info. 
```suggestion
    new_user_name = f"passbolt-py-user-%s@xyz.xyz"%(now.strftime("%Y-%m-%d-T%H%M%S"))
```